From e63cb27ab5f55cb1d9d238b46fb72f08011fc916 Mon Sep 17 00:00:00 2001
From: Junki Kato <junki.kato.xk@renesas.com>
Date: Tue, 11 Sep 2018 20:19:41 +0900
Subject: [PATCH 264/909] usb: host: xhci-rcar: slow speed device connect when
 suspend to RAM

commit e72602bee78ca9f11dd5be238cd7abd2fd83b3f0 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds slow speed device connect for not timeout
in handshake progress.

Signed-off-by: Junki Kato <junki.kato.xk@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/host/xhci-rcar.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/host/xhci-rcar.c b/drivers/usb/host/xhci-rcar.c
index 5d2fade16167..1f31f8461f8f 100644
--- a/drivers/usb/host/xhci-rcar.c
+++ b/drivers/usb/host/xhci-rcar.c
@@ -263,6 +263,7 @@ int xhci_rcar_init_quirk(struct usb_hcd *hcd)
 	if (!xhci_rcar_wait_for_pll_active(hcd))
 		return -ETIMEDOUT;
 
+	xhci->quirks |= XHCI_SLOW_SUSPEND;
 	xhci->quirks |= XHCI_TRUST_TX_LENGTH;
 	return xhci_rcar_download_firmware(hcd);
 }
-- 
2.17.1

